src/turnkeyml/run/basert.py

```diff
@@ -260,9 +260,9 @@ def throughput(self) -> float:
         Returns the throughput, in IPS, for the benchmarking run.
         """
 
-    @property
+    @staticmethod
     @abstractmethod
```

Comment:
Does it make sense to have those two decorators in here? Turns out it does!
TIL!
https://stackoverflow.com/questions/4474395/staticmethod-and-abc-abstractmethod-will-it-blend